lib/function/geometry/intersect.js

```diff
@@ -125,7 +125,7 @@ function factory (type, config, load, typed) {
   }
 
   function _intersectLinePlane(x1, y1, z1, x2, y2, z2, x, y, z, c){
-    var t = (c - x1*x - y1*y - z1*z)/(x2*x + y2*y + z2*z - x1 - y1 - z1);
+    var t = (c - x1*x - y1*y - z1*z)/(x2*x + y2*y + z2*z - x1*x - y1*y - z1*z);
```

Comment:
The correct derivation is as follows.

x * (x1 + t(x2 - x1)) + y *(y1 + t(y2 - y1)) + z * (z1 + t(z2 - z1)) = c
(x1 * x) + (x2 * x * t) - (x1 * x * t) + (y1 * y) + (y2 * y * t) - (y1 * y * t) + (z1 * z) + (z2 * z * t) - (z1 * z * t) = c
(x2 * x * t) - (x1 * x * t) + (y2 * y * t) - (y1 * y * t) + (z2 * z * t) - (z1 * z * t) = c - (x1 * x) - (y1 * y) - (z1 * z)
t * ((x2 * x) - (x1 * x) + (y2 * y) - (y1 * y) + (z2 * z) - (z1 * z)) = c - (x1 * x) - (y1 * y) - (z1 * z)
t = (c - (x1 * x) - (y1 * y) - (z1 * z)) / ((x2 * x) - (x1 * x) + (y2 * y) - (y1 * y) + (z2 * z) - (z1 * z))
t = (c - x1*x - y1 * y - z1*z) / (x2*x + y2*y + z2*z - x1*x - y1*y - z1*z)